tests/test_parser.py

```diff
@@ -40,6 +54,21 @@ def encoded_dataset(filename):
         return (encoder.content_type, encoder.to_string())
 
 
+class TestingValueTarget(ValueTarget):
+    def __init__(self):
+        super().__init__()
+        self.started = False
```

Comment:
Can this also be moved to the `Target` interface?

Reply:
But does Target users really need to have these states in Target?
I would say they are for unit tests only. Just to ensure how the library calls the Target.